lib/tile.js

```diff
@@ -158,29 +142,33 @@ Tile.prototype.containsPoint = function(x,y) {
 }
 
 Tile.prototype.sameTypeAs = function(tile) {
-	return this.type == tile.type;
+	if(tile) { 
+		return this.type == tile.type;
+	} else {
+		return false;
+	}
 }
 
 Tile.prototype.possibleMatch = function() {
-	if(this.tileBelow() && this.tileBelow().sameTypeAs(this)) {
```

Comment:
That's certainly a valid concern. I might improve it by passing it an object with xOffset and yOffset keys instead of straight numbers, maybe? Does that sound like a good idea?